### PR TITLE
Update information-barriers-policies.md

### PR DESCRIPTION
**Pull request by DPascua-MSFT**
Description of "Visibility of non-IB users and groups" has been corrected. If the difference between allow and block policies is a future release, please specify this as the previous explanation is not in current circulation of IB policies.
Trying to hyperlink "SharePoint Mode" to https://docs.microsoft.com/en-us/sharepoint/information-barriers#information-barriers-modes-and-sharepoint-sites
and hyperlink "OneDrive Mode" to https://docs.microsoft.com/en-us/onedrive/information-barriers#information-barriers-modes-and-onedrive

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/information-barriers-policies.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ When you define policies for IB, you'll work with several objects and concepts.
...
-- **Visibility of non-IB users and groups**. Non-IB users and groups are users and groups excluded from IB segments and policies. Depending on the type of IB policies (block or allow), the behavior for these users and group will differ in Microsoft Teams, SharePoint, OneDrive, and in your global address list. For users defined in *allow* policies, non-IB groups and users won't be visible to users included in IB segments and policies. For users defined in *block* policies, non-IB groups and users will be visible to users included in IB segments and policies.
+- **Visibility of non-IB users and groups**. Non-IB users and groups are users and groups excluded from IB segments and policies. Non-IB groups and users will be visible to Search in Teams and your global address list. SharePoint and OneDrive may allow search of non-IB groups and users depending on the [SharePoint Mode](/SharePoint/SharePointOnline/information-barriers.md#information-barriers-modes-and-sharepoint-sites) or [OneDrive Mode](/OneDrive/information-barriers.md#information-barriers-modes-and-onedrive).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/information-barriers-policies.md]:
```suggestion
- **Visibility of non-IB users and groups**. Non-IB users and groups are users and groups excluded from IB segments and policies. Non-IB groups and users will be visible to search in Teams and your global address list. SharePoint and OneDrive may allow search of non-IB groups and users depending on the [SharePoint Mode](/SharePoint/SharePointOnline/information-barriers#information-barriers-modes-and-sharepoint-sites) or [OneDrive Mode](/OneDrive/information-barriers#information-barriers-modes-and-onedrive).  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @DamienPascua, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by robmazz** [comment]
@DamienPascua - thanks for the feedback. This topic is undergoing changes for the newer version of IB, I will discuss this the PG and confirm that this addition is still accurate. Thanks again.

**Pull request closed without merging**